README.adoc

```diff
@@ -17,6 +17,55 @@ An end-to-end https://www.optaplanner.org/[OptaPlanner] web application ready to
 
 See https://www.youtube.com/watch?v=sOWC4qrXxFk[the video demo].
 
+== Quick Start
+
+Install Docker and run the following on the command line:
+
+[source,shell]
+----
+docker build -t rostering .
+docker run -p 8080:8080 --rm -it -e SPRING_PROFILES_ACTIVE=dev rostering
+----
+This takes a few minutes on the first time. The application will be automatically populated with test data.
+
+== Development
+
+
+=== Backend
+
+==== Set up OpenJDK Java 8
+
+[source,shell]
+----
+$ brew tap AdoptOpenJDK/openjdk
```

Comment:
I don't think we want to include install instructions for OpenJDK, especially since the install instructions differs between operating systems (for instance, brew is mostly specific to MacOS). Maybe include a link to the [OpenJDK install page](https://openjdk.java.net/install/) or [AdoptOpenJDK Installation Page](https://adoptopenjdk.net/installation.html)?

Reply:
I agree - I'll drop this and the addition of the bash profile instructions - it's fair to assume that the would-be developer is already familiar with developing in Java